contributed_definitions/NXunit_cell.nxdl.xml

```diff
@@ -21,79 +21,132 @@
+            <item value="tetragonal"/>
+            <item value="rhombohedral"/>
+            <item value="hexagonal"/>
+            <item value="cubic"/>
+        </enumeration>
+    </field>
+    <field name="space_group">
+        <doc>
+             Crystallographic space group.
+             A space group is the symmetry group of a repeating pattern in space.
+             For further information, see International Table for Crystallography (https://it.iucr.org/).
+        </doc>
+    </field>
+    <field name="point_group">
+        <doc>
+             Crystallographic point group.
+             A crystallographic point group is a set of symmetry operations, corresponding to one of the point groups in three dimensions,
+             such that each operation (perhaps followed by a translation) would leave the structure of a crystal unchanged.
+             This field should use Schoenflies notation (see Schoenflies, A., Krystallsysteme und Krystallstructur, 1891).
+             For further information, see https://dictionary.iucr.org/Point_group.
+        </doc>
+    </field>
+    <field name="laue_group">
+        <doc>
+             Laue group (also called Laue class).
+             The Laue classes are eleven geometric crystal classes containing centrosymmetric crystallographic types of point groups and their subgroups.
+             When absorption is negligible and Friedel's law applies, it is impossible to distinguish by diffraction between a centrosymmetric point group
+             and one of its non-centrosymmetric subgroups; only point groups belonging to different Laue classes can then be distinguished.
+             For further information, see https://dictionary.iucr.org/Laue_class.
         </doc>
     </field>
     <!--defined using which convention?-->
-    <field name="unit_cell_abc" type="NX_FLOAT" units="NX_LENGTH">
+    <field name="a_b_c" type="NX_FLOAT" units="NX_LENGTH">
         <doc>
-             Crystallography unit cell parameters a, b, and c.
+             Crystallography unit cell parameters a, b, and c
         </doc>
         <dimensions rank="1">
             <dim index="1" value="3"/>
         </dimensions>
     </field>
-    <!--defined using which convention?-->
-    <field name="unit_cell_alphabetagamma" type="NX_FLOAT" units="NX_ANGLE">
+    <field name="base_vector_a" type="NX_FLOAT" units="NX_LENGTH">
         <doc>
-             Crystallography unit cell parameters alpha, beta, and gamma.
+             Crystallography unit cell vector a
         </doc>
         <dimensions rank="1">
             <dim index="1" value="3"/>
         </dimensions>
+        <attribute name="depends_on">
+            <doc>
+                 For definining which coordinate system the unit cell vector a is defined in.
+            </doc>
+        </attribute>
     </field>
-    <field name="unit_cell_volume" type="NX_FLOAT" units="NX_VOLUME">
+    <field name="base_vector_b" type="NX_FLOAT" units="NX_LENGTH">
```

Comment:
but if the unit is length then this is the magnitude of the vector but not the components, but dimensions says it is a vector, i.e. that is the base vector in another coordinate system expressed in unit length hence Euclidean/real space, why not then better modelling this as NXtransformation ?